### DISCUSSION TITLE
Update existing google calendar entry

### DISCUSSION THREAD

**Discussion by jugglingcats** [add2cal/add-to-calendar-button#398] [Q&A] [answered]
I'm setting uid and sequence properties but get a new calendar entry created every time I add to Google Calendar or Outlook365 (with incrementing sequence number).

Just wondering if this is expected behaviour. I was hoping it would update the existing entry.

Thanks

**Comment by jekuer** [comment, marked as answer]
Unfortunately, they are not supporting this. At least not in a way, which would work with this open use case.
Updating existing events only works with iCal.
